landavailability/tests/webclient/test_views.py

```diff
@@ -165,7 +165,7 @@ def test_search_view_postcode(self):
             'nearest_broadband_fast': None
         }]
 
-        api_url = '{0}/api/locations/'.format(
+        api_url = '{0}/api/location-search'.format(
             settings.LAND_AVAILABILITY_API_URL)
```

Comment:
In case we merge the two methods (POST and GET) under the same view to bind them on /locations/ we should add a custom permission class like this:

```
from rest_framework.permissions import BasePermission, SAFE_METHODS
 
 
class IsAdminOrReadOnly(BasePermission):
    def has_permission(self, request, view):
        if request.method in SAFE_METHODS:
            return True
        else:
            return request.user.is_staff
```